reference/audit-logs.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Track activity in Axiom'
+description: "This page explains how to track activity in your Axiom organization with audit logs."
+sidebarTitle: Audit Logs
+tags:
+    ['audit logs', 'track activity']
+---
+
+Audit logs allow you to track who did what and when within your Axiom organization.
+
+Tracking activity in your Axiom organization with audit logs is useful for legal compliance reasons. For example, you can investigate the following:
+
+- Track who has accessed the Axiom platform.
+- Track organization access over time.
+- Track data access over time.
+
+Audit logs also make it easier to manage your Axiom organization. They allow you to do the following, among others:
+
+- Track changes made by your team to your observability posture.
+- Track monitoring performance.
+- Track billable usage.
```

Comment:
My bad for outdated Notion page, but this one has not made it in and will likely eventually be surfaced in a different dataset